index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+            <main id="main">
+                <header class="header">
+                     <H1 id="title">Dr. Norman Borlaug</H1>
+                     <p>The man who saved a billion lives</p>
+                </header>
```

Comment:
Semantic tag are good because a page made with semantic element is much easier to read.
Semantic tag offers a better user experience.

Reply:
- [ ] [OPTIONAL] Good job adding an explanation but you should start your comment with the issue in hand just like you did previously ..."I think it is better to use lowercase h1 rather than H1 for semantic reasons 👍 because Semantic tags are good because a page made with semantic element is much easier to read. The semantic tag offers a better user experience."